tests/unit/dataactvalidator/test_scripts_loadTas.py

```diff
@@ -88,3 +91,33 @@ def test_updateTASLookups(database, monkeypatch):
     assert results[1].agency_identifier == '1'  # replaces previous value
     assert results[2].tas_id == 333
     assert results[2].agency_identifier == '2'
+
+
+def test_add_start_date_blank(database):
+    """We backdate all of the TASLookups when there aren't any in the
+    database"""
+    assert database.session.query(TASLookup).count() == 0
+    data = pd.DataFrame({'dummy': ['data']})
+    loadTas.add_start_date(data)
+
+    assert data['internal_start_date'][0] == date(2015, 1, 1)
+
+
+@pytest.mark.parametrize("today_date,fiscal_date", [
+    ('2016-01-01', date(2016, 1, 1)),
+    ('2016-02-01', date(2016, 1, 1)),
+    ('2016-03-22', date(2016, 1, 1)),
+    ('2016-04-10', date(2016, 4, 1)),
+    ('2016-08-08', date(2016, 7, 1)),
+    ('2016-12-31', date(2016, 10, 1)),
+])
+def test_add_start_date_fiscal_year(database, today_date, fiscal_date):
+    """We should be calculating the correct beginning of a fiscal year"""
+    database.session.add(TASFactory())
+    database.session.commit()
+
+    data = pd.DataFrame({'dummy': ['data']})
+    with freeze_time(today_date):
```

Comment:
Traveling through time is also nifty. Thanks for continuing to introduce these helpful tools to the code base!